recipe/meta.yaml

```diff
@@ -153,20 +168,6 @@ requirements:
     - pytorch {{ version }} cuda{{ cuda_compiler_version | replace('.', '') }}_*_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
     - pytorch {{ version }} cpu_{{ blas_impl }}_*_{{ PKG_BUILDNUM }}                                 # [cuda_compiler_version == "None"]
 
-# these tests are for the libtorch output below, but due to
-# a particularity of conda-build, that output is defined in
-# the global build stage, including tests
-test:
-  commands:
-    # libraries
-    {% for each_lib in [ 'libc10', 'libshm', 'libtorch', 'libtorch_cpu', 'libtorch_global_deps'] %}
-    - test -f $PREFIX/lib/{{ each_lib }}.so     # [linux]
-    - test -f $PREFIX/lib/{{ each_lib }}.dylib  # [osx]
-    {% endfor %}
-    {% for each_lib in ['libc10_cuda', 'libcaffe2_nvrtc', 'libtorch_cuda', 'libtorch_cuda_linalg'] %}
-    - test -f $PREFIX/lib/{{ each_lib }}.so     # [linux and cuda_compiler_version != "None"]
-    {% endfor %}
```

Comment:
Do not delete these tests; add windows equivalents (you'll have to strip off the "lib" prefix off `each_lib`, because windows doesn't have that). Something like
```yaml
    {% for each_lib in [ 'c10', 'shm', 'torch', 'torch_cpu', 'torch_global_deps'] %}
    - test -f $PREFIX/lib/{{ each_lib }}.so     # [linux]
    - test -f $PREFIX/lib/{{ each_lib }}.dylib  # [osx]
    - if not exist %LIBRARY_BIN%\{{ each_lib }}.dll exit 1     # [win]
    - if not exist %LIBRARY_LIB%\{{ each_lib }}.lib exit 1     # [win]
    {% endfor %}
```
windows also seems to have `asmjit.dll` & `fbgemm.dll` (again, haven't checked unix logs)

Reply:
I added them back, don't know why they got removed. Will add the windows tests once we decide were to place them! Check my other comment.

Reply:
For these tests there's no question about placement or extra bloat; these should just be added.